### ISSUE TITLE
Zauberpuzzleanpassung

### ISSUE THREAD

**Issue by holidayfun** [fwtools/lightnoise#1]
Hey, 

könntet ihr bitte die Zauberpuzzleanpassung auch im lightnoise anschalten?
Wäre klasse! 

@kelunik 
@bwoebi 
@snigg

**Comment by ghost** [comment]
Meinst du die Runenpuzzleanpassung aus den *sunfire-Styles, sodass die Bilder durch Zahlen ersetzt werden? Oder welche Änderung meinst du?

**Comment by holidayfun** [comment]
Nee, ihr habt das Zauberpuzzlebild durch das hier ersetzt: https://github.com/fwtools/epicsunfire/blob/master/i/zauberpuzzle.png

das meine ich, das ist aber im lightnoise leider nicht enthalten
